README.md

```diff
@@ -49,7 +52,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town.delete(residents: "Belle")
+town.castle.new(guests: "Belle")
```

Comment:
Close, this would be syntax similar if we were working in Active Record, however here we are really accessing/ modifying key-value pairs in a hash. i.e.:
` town[:castle][:guests] << town[:residents][1]`